src/rpc/client/ts/src/settings.ts

```diff
@@ -20,20 +20,35 @@
 import { EditorClient } from './omega_edit_grpc_pb'
 import * as grpc from '@grpc/grpc-js'
 
-const port = process.env.OMEGA_EDIT_SERVER_PORT || '9000'
-const host = process.env.OMEGA_EDIT_SERVER_HOST || '127.0.0.1'
-const uri = process.env.OMEGA_EDIT_SERVER_URI || `${host}:${port}`
-
-let creds = grpc.credentials.createInsecure()
-const client = new EditorClient(uri, creds)
+var client: EditorClient | undefined = undefined
 export const NO_EVENTS = 0
 export const ALL_EVENTS = ~NO_EVENTS
 
 /**
- * Gets the connected editor client
+ * Initialize the editor client with the provided host and port
+ * @param host interface to connect to
+ * @param port port to bind to
+ */
+export function initClient(host: string = '127.0.0.1', port: string = '9000') {
+  const uri = process.env.OMEGA_EDIT_SERVER_URI || `${host}:${port}`
+
+  let creds = grpc.credentials.createInsecure()
```

Comment:
`let` can be `const` here, or just get rid of the temp variable and do `client = new EditorClient(uri, grpc.credentials.createInsecure())`.

Reply:
This method was supposed to be removed since I had the host and port to getClient instead, made it easer